docs/lib/Components/DropdownsPage.js

```diff
@@ -291,6 +293,86 @@ DropdownItem.propTypes = {
     <DropdownItem>Another Action</DropdownItem>
     <DropdownItem>Another Action</DropdownItem>
   </DropdownMenu>
+</Dropdown>`}
+          </PrismCode>
+        </pre>
+
+        <h3>Modifiers</h3>
+        <div className="docs-example">
+          <Row>
+            <Col>
+              <Dropdown>
```

Comment:
You need `isOpen` and `toggle` props, or you can use `UncontrolledDropdown` here (and just keep `Dropdown` with some dummy `isOpen` and `toggle` props in the raw example code which is displayed)

Reply:
omg, okay)